integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java

```diff
@@ -71,6 +72,7 @@ public void testSDXMultiRepairIDBRokerAndMasterWithTerminatedEC2Instances(TestCo
                     return testDto;
                 })
                 .awaitForDeletedInstancesOnProvider()
+                .await(SdxClusterStatusResponse.DELETED_ON_PROVIDER_SIDE)
```

Comment:
I think this is not necessary here, because of the previous `awaitForDeletedInstancesOnProvider` method.

Reply:
Yes, you are right, I removed it.